docs: add volcengine voice quickstart tutorial for EMQX AI

ysfscream opened the pull request:
Add comprehensive tutorial for building AI voice agent with EMQ + Volcengine RTC:
- Tutorial document with step-by-step setup instructions
- Architecture overview and component explanations
- Environment configuration guides (app, volc-server, web)
- Feature verification steps with screenshots placeholders
- Troubleshooting section for common issues
- Update dir.yaml to include new tutorial section

The tutorial demonstrates MCP over MQTT protocol implementation with:
- Real-time voice interaction via Volcengine RTC/ASR/TTS
- Device control capabilities (camera, expressions, volume)
- CustomLLM integration with AI Agent service

ysfscream committed:
docs: add volcengine voice quickstart tutorial for EMQX AI

Add comprehensive tutorial for building AI voice agent with EMQ + Volcengine RTC:
- Tutorial document with step-by-step setup instructions
- Architecture overview and component explanations
- Environment configuration guides (app, volc-server, web)
- Feature verification steps with screenshots placeholders
- Troubleshooting section for common issues
- Update dir.yaml to include new tutorial section

The tutorial demonstrates MCP over MQTT protocol implementation with:
- Real-time voice interaction via Volcengine RTC/ASR/TTS
- Device control capabilities (camera, expressions, volume)
- CustomLLM integration with AI Agent service

ysfscream committed:
docs: refine volcengine voice tutorial

- Remove TODO comments for screenshots
- Fix protocol naming: change "MQTT over MCP" to "MCP over MQTT"
- Clean up documentation formatting

jinfahua commented on zh_CN/emqx-ai/tutorials/quickstart-volcengine-voice.md:
```diff
@@ -0,0 +1,625 @@
+# 快速开始：基于 EMQ + 火山云语音服务搭建智能体
+
+本文档介绍如何使用 Docker Compose 快速部署一个支持语音交互与设备控制的 AI 智能体演示系统。本项目通过 PC 端的浏览器模拟智能设备端（摄像头、表情、音量等硬件能力），展示 MCP over MQTT 协议如何实现 AI Agent 对设备的实时控制。系统集成火山云 RTC 实现语音通道，ASR/TTS 提供语音识别与合成，CustomLLM 模式对接自定义 AI Agent 服务完成多轮对话与工具调用。
+
```
增加一个最终效果的链接，中文版本可以直接链接到我们在公众号上的视频录制

ysfscream replied:
找了下发现微信视频号无法分享 URL 链接，建议上传至 bilibili 和 Youtube 我们再更新上去

jinfahua replied:
已经有了的。
- https://www.bilibili.com/video/BV1P2WTzBEu4/?spm_id_from=888.80997.embed_other.whitelist&t=2.654235&bvid=BV1P2WTzBEu4

- https://youtu.be/x_RxJViZyLQ

ysfscream replied:
Updated

ysfscream committed:
docs: add English version of volcengine voice tutorial

Add English version of the quick start guide for building AI agents
with EMQ + Volcengine voice services. The English doc provides:
- Architecture overview and component descriptions
- Prerequisites summary with references to Chinese version for details
- Quick tutorial overview
- Troubleshooting reference

This resolves the CI failure caused by missing English translation.

ysfscream committed:
docs: add Japanese version for Volcengine voice quickstart tutorial

Add Japanese placeholder document for the EMQX AI tutorial, referencing the Chinese version for detailed instructions. This completes the multi-language requirement for CI validation.

ysfscream committed:
docs: address security review comments

- Simplify security warning for key generation
- Add security tip for .env files in git

ysfscream committed:
docs: add demo video links to volcengine voice tutorial

Meggielqk committed:
Editorial updates

Meggielqk committed:
Update dir.yaml

terry-xiaoyu committed:
docs: improve the first section in quickstart-volcengine-voice

terry-xiaoyu committed:
docs: refactor emqx-ai docs and add mcp-bridge

terry-xiaoyu committed:
docs: add kotlin docs

terry-xiaoyu committed:
remove en emqx-ai/overview.md

terry-xiaoyu committed:
Merge branch 'release-5.10' into mqtt-mcp-volo-rtc-ai-demo

ysfscream committed:
docs: fix broken links in volcengine-rtc docs

- Fix quickstart link path from non-existent file to ./quick-start.md
- Replace ./scenarios/ directory links with specific scenario page links

terry-xiaoyu committed:
chore: use cherry-studio instead of 5ire

jinfahua committed:
fix the links

ysfscream committed:
docs: improve volcengine RTC docs formatting

Meggielqk committed:
Add en translations

Meggielqk committed:
Update dir.yaml

Meggielqk committed:
Update dir.yaml

Meggielqk committed:
Update dir.yaml

terry-xiaoyu committed:
fix: change screenshot for en docs
